lib/gears/matcher.lua

```diff
@@ -381,7 +405,6 @@ end
 --
 -- @tparam string name The source name.
 -- @treturn boolean If the source has been removed.
--- @method remove_matching_source
```

Comment:
You only just now added all of these `@method` comments in a previous commit and now all of them are removed again?

Reply:
whoops, I copied changes between branch with different version of that pull request... fixed